_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
```

```suggestion
  description: Entwickle 2D- und 3D-Spiele, plattformübergreifende Projekte oder sogar XR-Ideen!
```
The *sogar* IMO ends up feeling a bit childish/ironic in german. YMMV. Personally, I'd go with something like the website title ("free and open source 2d and 3d engine") in german, also adding the XR bit. Feels more professional, dunno.

---

I tried to stick as close to the other translations as possible. Happy to change it though if there is consensus.

---

```suggestion
  description: Spielentwicklung in 2D & 3D, für platt­form­über­grei­fende Projekte sowie VR Ideen!
```

Taglines are some of the trickiest copy to localize, since marketing speech heavily varies based on what effect a certain sentence structure has in a given language. Starting with a verb in German feels odd/more childish as you described it.
Since I am to blame for the original in EN, I took the freedom to propose the vibe-equivalent here instead of a word for word transcript.